DNN Platform/DotNetNuke.Abstractions/Portals/IPortalInfo.cs

```diff
@@ -0,0 +1,310 @@
+
+        /// <summary>
+        /// Gets or sets payment Processor userId.
+        /// </summary>
+        /// <value>
+        string ProcessorUserId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the RoleId of the Registered users group of the portal.
+        /// </summary>
+        /// <value>RoleId of the Registered users. </value>
+        int RegisteredRoleId { get; set; }
+
+        /// <summary>
+        /// Gets or sets tabid of the Registration page.
+        /// </summary>
+        /// <value>TabId of the Registration page.</value>
+        int RegisterTabId { get; set; }
+
+        /// <summary>
+        /// Gets or sets tabid of the Search profile page.
+        /// </summary>
+        /// <value>TabdId of the Search Results page.</value>
+        int SearchTabId { get; set; }
+
+        /// <summary>
+        /// Gets or sets tabid of the Custom 404 page.
+        /// </summary>
+        /// <value>Tabid of the Custom 404 page.</value>
+        int Custom404TabId { get; set; }
+
+        /// <summary>
+        /// Gets or sets tabid of the Custom 500 error page.
+        /// </summary>
+        /// <value>Tabid of the Custom 500 error page.</value>
+        int Custom500TabId { get; set; }
+
+        /// <summary>
+        /// Gets or sets tabid of the Terms of Use page.
+        /// </summary>
+        /// <value>Tabid of the Terms of Use page.</value>
+        int TermsTabId { get; set; }
+
+        /// <summary>
+        /// Gets or sets tabid of the Privacy Statement page.
+        /// </summary>
+        /// <value>Tabid of the Privacy Statement page.</value>
+        int PrivacyTabId { get; set; }
+
+        /// <summary>
+        /// Gets or sets tabdId of the splash page. If 0, there is no splash page.
+        /// </summary>
+        /// <value>TabdId of the Splash page.</value>
+        int SplashTabId { get; set; }
+
+        /// <summary>
+        /// Gets or sets tabId at which Host tasks start.
+        /// </summary>
+        /// <value>TabId of Host tasks.</value>
+        int SuperTabId { get; set; }
+
+        /// <summary>
+        /// Gets or sets number of registered users allowed in the portal. 0 means not limited.
+        /// </summary>
+        /// <value>Number of registered users allowed. </value>
+        int UserQuota { get; set; }
+
+        /// <summary>
+        /// Gets or sets type of registration that the portal supports.
+        /// </summary>
+        /// <value>Type of registration.</value>
+        int UserRegistration { get; set; }
+
+        /// <summary>
+        /// Gets or sets tabid of the User profile page.
+        /// </summary>
+        /// <value>TabdId of the User profile page.</value>
+        int UserTabId { get; set; }
+
+        /// <summary>
+        /// Gets or sets actual number of actual users for this portal.
+        /// </summary>
+        /// <value>Number of users for the portal.</value>
+        int Users { get; set; }
```

Comment:
@mitchelsellers this seems like an opportunity to not include this property on the interface, in an effort to avoid requiring an `IPortalInfo` implementation to take a dependency on `IUserController` (or `IUserService` or whatever)

Reply:
Totally agree, there are other API's for this